docs/_layouts/default.html

```diff
@@ -24,12 +24,11 @@
     <nav class="sidebar">
       <ul class="nav-list">
         <li>
-          <a href="{{ '/' | relative_url }}">
-            <img src="{{ '/assets/img/bolt-logo.png' | relative_url }}" class="bolt-logo" />
-          </a>
-        </li>
-        <li>
-          <span class="nav-header">Hands-on Lab</span>
+          <span class="nav-header">
+            <a href="{{ '/' | relative_url }}">
+              <img src="{{ '/assets/images/bolt-logo.png' | relative_url }}" class="bolt-logo"> Bolt Hands-on Lab
```

Comment:
What does this markup render to? If possible, might want to add an alt attribute (I think this is purely decorative so, `alt=""` will make assistive technology ignore it.)

Reply:
Added this